tasks/48_follow-the-spy/follow-the-spy.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
+// I wanted to return flattened array, and first I use
+// Array.flat() method. My first solution was :
+// return [...new Set(result.flat())].join(', ')
+// It worked in JSBin but here I failed tests, as
+// "result.flat() is not a function". Strange.
+// So I flattened array using reduce method.
+
+export const followTheSpy = ( arr ) => {
+  const result = arr[0];
+  let foundRoute;
+
+  for ( let i = 1; i < arr.length; i++) {
+    if ( i === 1) {
```

Comment:
if you do something only for first index probably there is better way to do it
also you can add only last element of `foundRoute` ([1]) to result and there would be no reason after use concat, Set